packages/e2e-tests/specs/editor/various/font-size-picker.test.js

```diff
@@ -86,8 +86,14 @@ describe( 'Font Size Picker', () => {
 		);
 
 		await first( await page.$x( FONT_SIZE_LABEL_SELECTOR ) ).click();
+
```

Comment:
It feels like there is some sort of performance regression happening in the new component 🙁 Do you know why it takes longer than before?

Reply:
We don't. We may need to break out the react profiler for this one to check and see if there's something going on. There's no noticable regression in the UX, however, it's just from the perspective of the e2e tests.

Reply:
It's acceptable to add those delays when there is a need to wait for network requests or animations but otherwise it should work in the headless mode without any additional changes. I think that there are three things that could help:
- React profiler can validate if there isn't happening some longer processing than before
- React dev tools can identify unnecessary rerenders
- in e2e tests we disable animations when they are behind reduced motion guard, maybe it is missing

Reply:
Sounds good to me. @ItsJonQ and I can put our heads together on this one this week to figure it out.